H_ecto_models.md

```diff
@@ -621,11 +624,11 @@ end
 ```
 Now, when `Repo.update` is called for the `Video` model, the `approved_at` field will automatically be reset to nil. This should keep us safe from the ravages of an often hostile modern Internet!
 
-`Ecto.Model.Callbacks` actually ships with with a lot more than the `before_update` callback, in addition, it comes with:
+`Ecto.Model.Callbacks` actually ships with a lot more than the `before_update` callback, in addition, it comes with:
 
 - `before_delete`: called before the adapter deletes our record from the database; if perhaps we have a foreign_key constraint from another table, we would use this hook to clean it up.
 - `after_delete`: called after the adapter deletes our record. If we wished to archive our record, we'd do it in this hook.
-- `before_update`: called before the adapter updates our record. We used it above to reset the likes counter.
+- `before_update`: called before the adapter updates our record. We used it above to "null out" a video's `approved_at` field.
 - `after_update`: called after the adapter updates our record. If we wanted to update our user that the model has changed, now would do that here.
```

Comment:
I'm not sure what "If we wanted to update our user that the model has changed, now would do that here." is trying to say. Maybe something like "If we wanted to notify user that the model has changed, we would do that here."?